keras/losses.py

```diff
@@ -33,6 +33,11 @@ def hinge(y_true, y_pred):
     return K.mean(K.maximum(1. - y_true * y_pred, 0.), axis=-1)
 
 
+def logcosh(y_true, y_pred):
```

Comment:
logcosh seems to be the most standard naming i can find (log_cosh seems awkward and log-cosh is right out)